test/models/job_test.rb

```diff
@@ -253,14 +254,19 @@ def create
     end
 
     it "stops a queued job" do
-      active = stub(start!: true, on_complete: true, id: 123)
+      active_job = project.jobs.create!(command: 'cat foo', user: user, project: project, commit: 'master')
```

Comment:
I thought a best practice was to use `new` over `create` unless its quite inconvenient so that the tests run faster. In this test as far as I can see the object is only used in-memory. Trying to read it from the db again could be considered a performance bug. The main reason is so the tests run faster though.